src/web/web-service/frontend/index.html

```diff
@@ -199,27 +201,31 @@ <h4 style="margin: 18px 12px 12px 24px">OT Border Router</h4>
               <div layout="row">
                 <md-input-container flex="50">
                   <label>Network Key</label>
-                  <input required name="networkKey" ng-model="thread.networkKey">
+                  <input required minlength="32" maxlength="32" ng-pattern="/^[0-9a-fA-F]{32}/g" name="networkKey" ng-model="thread.networkKey">
                   <div ng-messages="threadForm.networkKey.$error">
-                    <div ng-message="required">This is required.</div>
+                    <div ng-message-exp="['required', 'minlength', 'maxlength', 'pattern']">
+                      Network Key must be 32 hexadecimal digits long.
+                    </div>
                   </div>
                 </md-input-container>
 
                 <md-input-container flex="50">
                   <label>Channel</label>
-                  <input required name="channel" ng-model="thread.channel">
+                  <input required name="channel" type="number" min="11" max="26" ng-model="thread.channel">
                   <div ng-messages="threadForm.channel.$error">
-                    <div ng-message="required">This is required.</div>
+                    <div ng-message-exp="['required', 'min', 'max']">Channel must be in 11~26.</div>
                   </div>
                 </md-input-container>
               </div>
 
               <div layout="row">
                 <md-input-container flex="100">
                   <label>On-Mesh Prefix</label>
-                  <input required name="prefix" ng-model="thread.prefix">
+                  <input required name="prefix" ng-model="thread.prefix" ng-pattern="/^(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1})/g">
```

Comment:
Does this regex allows prefix length?
Is link-local address allowed as Thread prefix?